modules/monitoring-configuring-a-local-persistent-volume-claim.adoc

```diff
@@ -1,25 +1,34 @@
 // Module included in the following assemblies:
 //
-// * monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc
+// * monitoring/configuring-the-monitoring-stack.adoc
 
 [id="configuring-a-local-persistent-volume-claim_{context}"]
 = Configuring a local persistent volume claim
 
-For the Prometheus or Alertmanager to use a persistent volume (PV), you first must configure a persistent volume claim (PVC).
+For monitoring components to use a persistent volume (PV), you must first configure a persistent volume claim (PVC).
```

Comment:
the doc has the steps to configure PVCs
https://github.com/openshift/openshift-docs/blob/3c8ccacfc4d60ef106a5d31f29a65ad45076f738/modules/monitoring-configuring-a-local-persistent-volume-claim.adoc

Reply:
@lihongyan1 thank you for your comment. This procedure module provides steps to configure a PVC. I have removed the word 'first' so that this is more obvious.